cli_docs/generate_cli_docs.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+PYTHONPATH=.. python3 generate_cli_docs.py
```

Comment:
you need to do 
```
cd `dirname $0`
```
or something like this, because the meaning of this script is to properly setup python path, which will not properly work if you do not set working directory

Reply:
Here is an example
```
antban@Zalando-10567:~/workspace/zalando/bubuku/cli_docs$ cat tmp.sh 
#/bin/bash
echo `dirname $0`
antban@Zalando-10567:~/workspace/zalando/bubuku/cli_docs$ ./tmp.sh 
.
antban@Zalando-10567:~/workspace/zalando/bubuku/cli_docs$ cd ..
antban@Zalando-10567:~/workspace/zalando/bubuku$ ./cli_docs/tmp.sh 
./cli_docs
```

Reply:
Makes sense. Fixed https://github.com/zalando-nakadi/bubuku/pull/137/commits/9727468909362b8542eb36b864756c02ed11536b